public/resources/js/comments.js

```diff
@@ -95,9 +97,18 @@ $(document).ready(function() {
         return html;
     }
 
-    function generateCommentHTML(comment) {
+    function generateCommentsHTML(comments) {
+        let html = '';
+        const parentComments = comments.filter(comment => comment.parent_comment_id === null);
+        parentComments.forEach(parentComment => {
+            html += generateCommentHTML(parentComment, comments);
+        });
+        return html;
+    }
+    
+    function generateCommentHTML(comment, allComments, level = 0) {
         let html = `
-            <div class="comment" style="margin-left: ${comment.level * 5}px;">
+            <div class="comment" style="margin-left: ${level * 5}px;">
```

Comment:
make a note, or talk with the front end team to review this and correct, because it is bad practice to make css on html